packages/core/kind/debugClassnames.js

```diff
@@ -0,0 +1,53 @@
+/* !
+  classnames Copyright (c) 2018 Jed Watson.
+  Licensed under the MIT License (MIT), see
+  http://jedwatson.github.io/classnames
+*/
+
+const hasProp = {}.hasOwnProperty;
+
+// When passed as strings (not in objects!), these strings are acceptable globals
+const allowedGlobalClasses = ['neutral', 'highContrast', 'spottable', 'enact-fit'];
```

Comment:
As you note in your description, since these are sandstone-specific, they will be problematic to be declared in `core`.

Reply:
Correct. These are skinning specific and spotlight specific (enact-fit is fine). This would need to at least be design-time configurable so they can be set up in the ThemeDecorator.